src/typings/index.d.ts

```diff
@@ -32,5 +32,8 @@ declare module "@expo/samples" {
 // getting an error on my end. hoping to clear it up. sL
 declare module "@expo/vector-icons";
 
+// Getting an error, pls work
+declare module "react-native-vertical-swipe";
```

Comment:
Obviously, I assume this will be fixed after the demo, right? :p

Reply:
That was cody :P

Reply:
I'm callin u out @Kyrk